src/Error.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Error.css';
+
+const Error = ({ errorMessage }) => {
+    return (
+        <h2>{errorMessage}</h2>
+    );
+};
+
+export default Error;
```

Comment:
You might be planning on doing more with this Error component in the future, but as it is now, having just one h2 for a component makes me think it probably doesn't need to be a separate component.